AppBundles/CreateThumbnailPlugin/CreateThumbnailPlugin.csproj

```diff
@@ -34,6 +34,9 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Autodesk.Forge.DesignAutomation.Inventor.Utils, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Autodesk.Forge.DesignAutomation.Inventor.Utils.2.0.0\lib\netstandard2.0\Autodesk.Forge.DesignAutomation.Inventor.Utils.dll</HintPath>
```

Comment:
This is wrong.  You should reference it  using the $(InventorBinFolder) like here https://github.com/Autodesk-Forge/forge-configurator-inventor/pull/374/files#diff-2b9ecf57b5ae66a58fe796827d30f97e4de1e5c326a98044c7980f2b5b9f13dbR46